book/README.md

```diff
@@ -1,48 +1,47 @@
 # TLG Catalog
 
-Catalog for tables, listings, and graphs generated from NEST R packages.
+The catalog for (T)ables, (L)istings, and (G)raphs for clinical trials generated using NEST R packages.
 
-## Architecture
+The main goal of this repository is to provide a comprehensive collection of clinical trials outputs generated using R packages. The target audience is the clinical trials community, including statisticians, data scientists, and other professionals interested in applying R to clinical trials data.
 
-This repository follows a very straightforward architecture.
+## Usage
 
-``` mermaid
-sequenceDiagram
-    autonumber
-    Author->>github.com: Creates content + pushes updates
-    github.com->>Github Pages: Builds and publishes
-    Analyst->>Github Pages: Refers to catalog
-```
+Each of the output is generated in separate articicle and usually consists of:
 
-## Technology
+* Data setup using dummy, synthetic data.
+* Output itself (of many variants if applicable).
+* Interactive application.
+* Reproducibility information.
 
-- Dependency management: [`staged.dependencies`](https://openpharma.github.io/staged.dependencies/) for installing dependencies
-- Building and rendering: [`quarto`](https://quarto.org/)
-- Publishing: [`Github Pages`](https://pages.github.com/) hosted at <https://insightsengineering.github.io/tlg-catalog>
+See the full list of available TLGs in [Index page](tlg-index.qmd).
```

Comment:
It took me a while to check it but I can confirm that from within the webpage it would redirect to the subpage and not GH. From within GH it will be GH. This looks correct.